### PR TITLE
Update dat_fukken_disk.yml

### PR DESCRIPTION
**Pull request by GoodWheatley**
Changed nuke disk and fake disk size to 5, because I don’t think it makes sense that a laser gun takes up less space than a floppy disk

## About the PR 

**Screenshots**

**Changelog**
<!--
Here you can fill out a changelog that will automatically be added to the game when your PR is merged
There are 4 icons for changelog entries: add, remove, tweak, fix. I trust you can figure out the rest.

You can put your name after the :cl: symbol to change the name that shows in the changelog (otherwise it takes your GitHub username)
Like so: :cl: PJB

Generally, only put things in changelogs that players actually care about. Stuff like "Refactored X system, no changes should be visible" shouldn't be on a changelog.

For writing actual entries, don't consider the entry type suffix (e.g. add) to be "part" of the sentence:
bad: - add: a new tool for engineers
good: - add: added a new tool for engineers
--
:cl:
- tweak: nuke disk now takes up 5 space instead of 12

### CODE REVIEW

**Review by Peptide90 — APPROVED** [review]
Looks good to me.

**Comment by Peptide90** [comment]
Scratch that. You can actually just remove the item component all together and it’ll inherit size 5 from the parent.

**Comment by GoodWheatley** [comment]
Ok, how do I do that?

**Comment by GoodWheatley** [comment]
Alright after failing to figure this out, merging it in the current state it’s in won’t cause any problems, right?

File: Resources/Prototypes/Entities/Objects/Misc/dat_fukken_disk.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@
     sprite: Objects/Misc/nukedisk.rsi
     state: icon
   - type: Item
-    size: 12
```
Comment (Peptide90) [inline Resources/Prototypes/Entities/Objects/Misc/dat_fukken_disk.yml:13]: So lines 12 to 15 here are your item component. Delete them (everything up to the next component defined with the -)

File: Resources/Prototypes/Entities/Objects/Misc/dat_fukken_disk.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@
...
```
Comment (Peptide90) [inline Resources/Prototypes/Entities/Objects/Misc/dat_fukken_disk.yml:31]: Same here, lines 30 to 33 delete.

**Review by Peptide90 — CHANGES_REQUESTED** [review]
Pointed out the bits I’d say to remove.

Otherwise if that’s not ideal then just remove the line that has size, that way it inherits size from the parent but the item component is still specifying the item sprite (this is in sprite component too so not sure it’s necessary)

**Comment by GoodWheatley** [comment]
sorry, this is my first time using github, is there any way to accept the changes you've suggested?

**Comment by ElectroJr** [comment]
You can just make the changes to your local branch, and then push them to your github. the PR will automatically update

**Comment by GoodWheatley** [comment]
This is way more stressful than I thought it would be

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**